docs/types.rst

```diff
@@ -460,11 +460,17 @@ a non-rational number).
 .. index:: literal, literal;string, string
 .. _string_literals:
 
-String Literals
----------------
+String Literals and Types
+-------------------------
 
 String literals are written with either double or single-quotes (``"foo"`` or ``'bar'``).  They do not imply trailing zeroes as in C; ``"foo"`` represents three bytes, not four.  As with integer literals, their type can vary, but they are implicitly convertible to ``bytes1``, ..., ``bytes32``, if they fit, to ``bytes`` and to ``string``.
 
+For example, with ``"stringliteral"`` the string literal is interpreted in its raw byte form and with certain tools, if you inspect it, you see a 32-byte hex value, which is the ``"stringliteral"`` in hex.
+
+A string literal becomes a ``string`` type when you assign it to a variable, for example::
```

Comment:
Isn't the comment above ("can be converted") the same as here? Also note that in its current form, it is incorrect, because it says `string` here and `bytes32` below.

Reply:
They are slightly different in explanation, but now the above is deleted anyway (https://github.com/ethereum/solidity/pull/5437/files#r236217406) I think the duplication is gone.

Reply:
Ah, first I realised that the FAQ point matching this hadn't been removed in this new PR, and rereading that I clarified this text to make sense. But it may be now that the addition isn't so valuable.